docs/source/gateway/index.rst

```diff
@@ -1,6 +1,1049 @@
+
+.. code-block:: python
+
+    import requests
+
+    url = "https://<your workspace>.databricks.com/api/2.0/gateway/routes"
+
+    data = {
+        "name": "my-completions",
+        "route_type": "llm/v1/completions",
+        "model": {
+            "name": "text-davinci-003",
+            "provider": "openai",
+            "openai_config": {"openai_api_key": "<redacted secret key>"},
+        },
+    }
+
+    response = requests.post(url, headers=headers, json=data)
+
+* ``DELETE /api/2.0/gateway/routes`` **Databricks-only API** [DeleteRoute]: This endpoint deletes a route from the AI Gateway service based on the provided name.
+
+An example usage of the ``DeleteRoute`` API in Databricks is:
+
+.. code-block:: python
+
+    import requests
+
+    url = "https://<your workspace>.databricks.com/api/2.0/gateway/routes"
+
+    data = {
+        "name": "my-completions",
+    }
+
+    requests.delete(url, headers=headers, json=data)
+
+
+Using these endpoints, you can interact directly with the Gateway Server from any platform or language
+that supports HTTP, providing a powerful and flexible way to leverage the functionality of the MLflow AI Gateway.
+
+Examples of Get Requests
+~~~~~~~~~~~~~~~~~~~~~~~~
+You can use the GET requests to retrieve information about the routes. The routes endpoint is
+located at "/gateway/routes". You can retrieve information about all routes or a specific route by
+appending the route name to the end of the URL.
+
+Get all routes:
+
+.. code-block:: bash
+
+    curl -X GET http://my.gateway:9000/gateway/routes
+
+This command will return a JSON object containing all routes currently configured on the Gateway Server.
+
+
+If you know the name of the route you're interested in, you can append it to the routes endpoint.
+For example, to get information about the route named ``completions``, you would use the following command:
+
+.. code-block:: bash
+
+    curl -X GET http://my.gateway:9000/gateway/routes/completions
+
+This command will return a JSON object with the configuration details for the completions route.
+
+Examples of Post Requests
+~~~~~~~~~~~~~~~~~~~~~~~~~
+You can use the POST request to send a query to a specific route.
+To send a query to a specific route, append the route name to the routes endpoint, and include the
+data to be sent in the body of the request. The structure of this data will depend on the specific model the route is configured for.
+
+For instance, to send a query to the completions route, you might use the following command:
+
+.. code-block:: bash
+
+    curl -X POST -H "Content-Type: application/json" -d '{"prompt": "It is a truth universally acknowledged"}' http://my.gateway:9000/gateway/routes/completions
+
+This will return a JSON object with the response from the completions model, which is usually the continuation of the text provided as a prompt.
+
+**Note:** Please remember to replace "http://my.gateway:9000" with the URL of your actual Gateway Server.
+
+Client API
```

Comment:
Can we combine the `Client API` and `Fluent API` sections into an `MLflow Python Client APIs` section that includes a *brief* explanation of what the fluent & client APIs are and shows *one* example API call for fluent and *one* for client?